src/components/PredictionGame.tsx

```
@@ -365,7 +380,19 @@ const AssignmentPrediction: FC<AssignmentPredictionProps> = ({
       {isCollapsed ? (
         <div
           className="flex cursor-pointer items-center gap-2 rounded-lg border border-gray-700/50 bg-gray-800/40 p-2 transition-colors hover:bg-gray-800/60 sm:gap-4 sm:p-3"
-          onClick={() => setUserExpanded(true)}
+          role="button"
```

Biome (2.4.7)

[error] The elements with this role can be changed to the following elements:
<button>

(lint/a11y/useSemanticElements)